### PR TITLE
feat: update codebase

### PR DESCRIPTION
**Pull request by johnny243**
- switch to `yarn`
- update deps
- upgrade to `component-relay@2.0.3`
- switch to `webpack`
- lint codebase

### CODE REVIEW

**Review by atmoio — APPROVED** [review]

File: Gruntfile.js [hunk 1]
```diff
@@ -1,120 +0,0 @@
...
-        dest: 'dist/app.js',
-      },
-
-      lib: {
-        src: [
-          'node_modules/jquery/dist/jquery.min.js',
-          'node_modules/bootstrap/dist/js/bootstrap.min.js',
-          'node_modules/summernote/dist/summernote.min.js',
-          'node_modules/sn-components-api/dist/dist.js',
-          'vendor/modes/markdown/markdown.js'
```
Comment (johnny243) [inline Gruntfile.js:71]: @mobitar `vendor` directory did not exist before this PR, but I'm wondering if this is the `CodeMirror` plugin for Markdown: https://codemirror.net/mode/markdown/

Comment (atmoio) [reply Gruntfile.js:71]: Could have been a copy/paste from another editor repo..

Comment (atmoio) [reply Gruntfile.js:71]: (I don't think this repo has anything to do with Markdown)

Comment (johnny243) [reply Gruntfile.js:71]: That's what I thought, thanks 😄

**Pull request merged**